src/post_section/post_section.jsx

```diff
@@ -30,102 +31,63 @@ function LinksList({ link }) {
         alt={link.name} /></Link>
     </td>;
 }
-class PostSection extends React.Component {
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        return (
-            <div>
-                <section className="second_section">
-                    <div className="second_section_block">
-                        <div className="author_of_post_info">
-                            <div className="inf_about_author">
-                                <img src="/photos/avatar.png" width="56" height="56" alt="avatar" />
-                                <div className="text_about_author">
-                                    <p className="author_name">Mika Matikainen</p>
-                                    <p className="time"> <time datetime="2020-04-15">Apr 15, 2020</time> · 4 min read</p>
-                                </div>
+const PostSection = () => {
+
+    const [data, setPosts] = useState({ posts: [] })
```

Comment:
why do you use array as a default value for a single post? shouldn't it be `{ post: {} }`?